### ISSUE TITLE
where should i start to debug this model?

### ISSUE THREAD

**Issue by BowieHsu** [Vladkryvoruchko/PSPNet-Keras-tensorflow#1]
Hi, @Vladkryvoruchko , Thank you for your hard work,
I am very interested in your work about convert PSPNet model from caffe to keras,
after run current model successfully , I also confused about the results
first of all, can you try to make sure the convert process is really worked?

**Comment by Vladkryvoruchko** [comment]
Hi @BowieHsu i will try to make backward converting to the binaryproto file(caffemodel) and check if npy is not corrupted after **caffe-tensorflow**.
Also will have a detailed view to the assigning weights and biases in keras by name.
Maybe something wrong with converting of batch normalization. Also will have to check

**Comment by hellochick** [comment]
hI, @Vladkryvoruchko could you tell me what you did to debug this origin model? Is something wrong with batch normalization? Thank you!
